Makefile

```diff
@@ -356,7 +356,13 @@ smoketest:
 	@$(MD5SUM) test.hex
 	$(TINYGO) build -size short -o test.hex -target=arduino-nano33      examples/blinky1
 	@$(MD5SUM) test.hex
-	$(TINYGO) build -size short -o test.hex -target=pico       			examples/blinky1
+	$(TINYGO) build -size short -o test.hex -target=pico                examples/blinky1
+	@$(MD5SUM) test.hex
+	$(TINYGO) build -size short -o test.hex -target=nano-33-ble         examples/blinky1
+	@$(MD5SUM) test.hex
+	$(TINYGO) build -size short -o test.hex -target=nano-33-ble-sense   examples/blinky1
+	@$(MD5SUM) test.hex
+	$(TINYGO) build -size short -o test.hex -target=nano-rp2040         examples/blinky1
```

Comment:
Honestly I don't think we need smoke tests for aliases. They only make the CI longer and the only thing they test is that the alias works (which it should, as long as the JSON file isn't modified).

Reply:
Sure, depends what we want to test with smokes (I can imagine json become corrupted after, say, some refactoring campaign).
I remove this then, unless @sago35 or @deadprogram have anything to say?

Reply:
:scissors: